pkg/skaffold/runner/load_images.go

```diff
@@ -48,6 +48,22 @@ func (r *SkaffoldRunner) loadImagesInK3dNodes(ctx context.Context, out io.Writer
 	})
 }
 
+// loadImagesInMicrok8sNodes loads artifact images into every node of a microk8s cluster.
+func (r *SkaffoldRunner) loadImagesInMicrok8sNodes(ctx context.Context, out io.Writer, k8sCluster string, artifacts []build.Artifact) error {
+	color.Default.Fprintln(out, "Loading images into microk8s cluster nodes...")
+	return r.loadImages(ctx, out, artifacts, func(tag string) *exec.Cmd {
+		// Note: needs multipass installed and a vm of name 'microk8s-vm' -- how to ensure?
+
+		// 1. build the image
+		// docker save image-name | multipass transfer - microk8s-vm:image-name.tar
+		exec.CommandContext(ctx, "docker", "save", "image-name", "|", "multipass", "transfer", "-", "microk8s-vm:image-name.tar")
```

Comment:
the `image_name` should correspond to the image built by docker right?
shdn't this be the image name in the `tag` field without the tag?